src/notte_eval/screenshots.py

```diff
@@ -0,0 +1,103 @@
+import base64
```

Comment:
this module can be removed after PR of last week (cf `notte.utils` or something like that)